### DISCUSSION TITLE
Config: placeholder for a dynamic value with a EnumNode.

### DISCUSSION THREAD

**Discussion by ecourtial** [symfony/symfony#45588] [Q&A] [answered]
In the _EnumNode_ of the _Config_ component, we see that it does not allow the use of a placeholder : https://github.com/symfony/symfony/blob/aa51dc73e627397ad0d2357e7ac76ff686fd4ffa/src/Symfony/Component/Config/Definition/EnumNode.php#L63

This is strange, because we could have some use cases where we would like to be able to define the value with a env variable. A good example is the samesite policy for the session cookie :  https://symfony.com/blog/new-in-symfony-4-2-samesite-cookie-configuration

Does anyone has ever faced this issue? Or is there another way to proceed, a workaround ?

**Comment by lyrixx** [comment, marked as answer]
Hello. indeed this seems a limitation.

Do you want to try to submit a PR to fix that?

**Reply by ecourtial** [reply]
>Do you want to try to submit a PR to fix that?

OK I will do that.

**Comment by ecourtial** [comment]
See https://github.com/symfony/symfony/pull/45624
